foundation-models/openai/src/main/java/com/sap/ai/sdk/foundationmodels/openai/OpenAiClient.java

```diff
@@ -137,23 +153,54 @@ public OpenAiChatCompletionOutput chatCompletion(@Nonnull final String prompt)
   }
 
   /**
-   * Generate a completion for the given prompt.
+   * Generate a completion for the given conversation and request parameters.
    *
-   * @param parameters the prompt, including messages and other parameters.
+   * @param request the completion request.
    * @return the completion output
    * @throws OpenAiClientException if the request fails
    */
   @Nonnull
+  public OpenAiChatCompletionResponse chatCompletion(
```

Comment:
(Minor)

This is new API right? Please add `@since [...]` to the JavaDoc. Applies to all new public API.